awscli/customizations/generatecliskeleton.py

```diff
@@ -36,18 +40,20 @@ class GenerateCliSkeletonArgument(OverrideRequiredArgsArgument):
 
     The argument, if present in the command line, will prevent the intended
     command from taking place. Instead, it will generate a JSON skeleton and
-    print it to standard output. This JSON skeleton then can be filled out
-    and can be used as input to ``--input-cli-json`` in order to run the
-    command with the filled out JSON skeleton.
+    print it to standard output.
     """
     ARG_DATA = {
         'name': 'generate-cli-skeleton',
-        'help_text': 'Prints a sample input JSON to standard output. Note the '
-                     'specified operation is not run if this argument is '
-                     'specified. The sample input can be used as an argument '
-                     'for ``--cli-input-json``.',
-        'action': 'store_true',
-        'group_name': 'generate_cli_skeleton'
+        'help_text': (
+            'Prints a JSON skeleton to standard output without sending '
+            'an API request. If provided with no value or the value '
+            '``input``, prints a sample input JSON that can be used as an '
+            'argument for ``--cli-input-json``. If provided with the value '
+            '``output``, it validates the command inputs and returns a '
+            'sample output JSON for that command.'
+        ),
+        'nargs': '?', 'const': 'input',
```

Comment:
I missed the `const: input` part initially, can you move this to a separate line?